test/unit/lookup.test.js

```diff
@@ -22,43 +22,64 @@ describe('lookup action', function () {
     });
   });
   describe('happy path', function () {
-    let revert_request_mock, request_spy; // stubbing request module
+    let revert_lookup_agreement_mock, lookup_agreement_spy;
     let revert_parse_mock, parse_spy; // stubbing csv-parse module
     let revert_access_token_mock;
+    let response_mock;
     beforeEach(function () {
       revert_access_token_mock = spyOn(utils, 'get_adobe_sign_access_token').and.returnValue(Promise.resolve({ access_token: 'gimme ze access codes!' }));
-      request_spy = jasmine.createSpy('request spy').and.callFake(function (options) {
-        if (options.url.includes('oauth/refresh')) {
```

Comment:
This mock was not being called for an access token, because we're already mocking `utils.get_adobe_sign_access_token`. 
I tend to avoid mocking generic methods like `request`, so I broke down `lookupSingleAgreement` and mocked `lookupAgreement` instead.